### PR TITLE
Creating routine to access mongodb

### PR DESCRIPTION
**Pull request by brenoaraujolima**
The access database can be made filling the `.env` file this way: [dot-env](https://gitlab.com/SoftSite/geosales/-/issues/493#note_1397714692)

### CODE REVIEW

**Review by jeffque — CHANGES_REQUESTED** [review]
This should not be dependent on #29, but the other way around

**Comment by brenoaraujolima** [comment]
> This should not be dependent on #29, but the other way around

Rebased on main.

File: .env.example [hunk 1]
```diff
@@ -1 +1,5 @@
-URL_AUTH_SERVICE=
\ No newline at end of file
+URL_AUTH_SERVICE=
+DB_SERVER=
```
Comment (brenoaraujolima) [inline .env.example:2]: These variables need to be created as a secret in repository's settings.

File: .env.example [hunk 2]
```diff
@@ -1 +1,5 @@
...
+DB_USER=
```
Comment (brenoaraujolima) [inline .env.example:3]: Put these variables in CI jobs

**Review by uxjp — APPROVED** [review]

**Review by williamvb — APPROVED** [review]

**Pull request merged**